src/main.ts

```diff
@@ -121,6 +131,8 @@ const plugin = function install(Vue: typeof _Vue, options?: CmsOptions) {
     },
   });
 
+  addDirectives(Vue, pluginOptions.trackClickHandler);
```

Comment:
If doing this, seems better to just pass all the options and use them as necessary.

```
addDirectives(Vue, pluginOptions);
```

Reply:
good point.